tap_snowflake/client.py

```diff
@@ -162,7 +162,10 @@ def create_engine(self) -> sqlalchemy.engine.Engine:
         Returns:
             A SQLAlchemy engine.
         """
-        connect_args = {}
+        connect_args: dict[str, Any] = {
+            "client_request_mfa_token": True,
+            "client_store_temporary_credential": True,
+        }
```

Comment:
Is adding these here safe for other auth methods? i.e. does is not affect the functioning of, for example, key-pair authentication?

Reply:
Actually the only auth method I did not test was the key pair authentication, but I can do an extra test with it. But I've been using these parameters with basic authentication, basic auth + mfa and browser authentication for a while and never had any issues. Will do an extra test with key pair auth just in case, but overall do not expect it to be an issue.

Reply:
Thanks @dlouseiro 🙏

Reply:
@edgarrmondragon , all good! It works fine with all auth methods.

Reply:
Thanks for confirming @dlouseiro!